Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:14.04
+
+RUN apt-get update && \
+    apt-get install -y wget \
+                       curl \
+                       software-properties-common \
+                       git \
```

Comment:
This is not needed (see comment below)

Reply:
the underlying code uses git. maybe we should change that.